src/plotSearchAndCompetitionsPage/components/mapSearchComponent.tsx

```diff
@@ -281,18 +310,21 @@ const MapSearchComponent = ({
                           <div role="list">
                             {section.targets.map((target) => (
                               <Row
-                                className="MapSearchComponent__target"
+                                className={classNames(
+                                  'MapSearchComponent__target',
+                                  {
+                                    MapSearchComponent__target__favourited:
```

Comment:
This is a modifier in BEM terminology; i.e. a `MapSearchComponent__target` that has an optional additional state. So: `MapSearchComponent__target--favourited` .